src/platforms/javascript/guides/capacitor/troubleshooting.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Troubleshooting
+description: "Troubleshoot and resolve common issues with the Capacitor SDK."
+sidebar_order: 1000
+---
+
+## Using with Capacitor 2 on iOS
+
+Because Capacitor 3 has a minimum requirement of iOS 12.0, the Sentry SDK needs to match this requirement to support Capacitor 3. Sadly, users on older versions my run into an error such as
```

Comment:
```suggestion
Because Capacitor 3 has a minimum requirement of iOS 12.0, the Sentry SDK needs to match this requirement to support Capacitor 3. Users on older versions may run into the following error:
```